angular-universal-ssr/e2e/methods/methods.ts

```diff
@@ -29,7 +29,7 @@ export class AngularUniversalSsrMethods extends BaseMethods {
             addedCities.forEach((city: string, counter: number) => {
                 cy.get(addedCitySelector).contains(city).click()
                 cy.get(selectedCityInfoSelector).contains(selectedCityInfo[counter]).should('be.visible')
-                this.reloadWindow()
+                cy.reload(true)
```

Comment:
why?

Reply:
Cause this keyword doesn't work in cy.origin for some reason. It works earlier?  But i didn't know how( You can check it on master now